src/autocomplete.ts

```diff
@@ -260,12 +260,10 @@ function createInputDropdown<State>({
             : findAll(config.dropdownSelector, HTMLElement)
 
     if (dropdownElements.length === 0) {
-        console.error(`No dropdown element found for input ${input}`)
+        log(new Error(`No dropdown element found for input ${input}`), "error")
```

Comment:
I would use Error instances only when something is available from the context (try/catch etc)